CHANGELOG.md

```
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## \[[0.3.4](https://github.com///compare/v0.3.3...v0.3.4)\] - 2025-11-13
+
+### Features
+
+- Add basic OTLP metics to the sbd-server that can be exported to a metrics server with the new CLI flag `--otlp-endpoint`
+
+### Build System
+
+- Pin hyper to 1.7.0 to workaround upstream issue where axum-server fails to compile. See https://github.com/programatik29/axum-server/issues/170 (#65)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)